index.html

```diff
@@ -0,0 +1,90 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Survey Form</title>
+    <link rel="stylesheet" href="/style.css">
```

Comment:
It would be great if you can change "/style.css" to either "./style.css"  or "style.css" because when you open the index file outside of VS code, it doesn't render the styling as shown in the picture below.
![checking](https://user-images.githubusercontent.com/94363151/205600134-228f2e54-5bf5-4c12-8a77-c6f0f3a11b6c.PNG)